data/other/logs/package/history.yml

```diff
@@ -0,0 +1 @@
+log_path: /var/log/apt/history.log
```

Comment:
Can we add also `term.log` which is really the actual log of apt operations ?

Reply:
Good idea